src/lib/Footer.svelte

```diff
@@ -15,9 +15,15 @@
 		</div>
 	{/if}
 	<div class="footer primary-bg">
+		<a href="https://bsky.app/profile/dddsouthwest.bsky.social" target="_blank">
+			<i class="fa-brands fa-bluesky fa-3x" title="DDD South West BlueSky"></i>
+		</a>
 		<a href="https://twitter.com/dddsouthwest" target="_blank">
 			<i class="fa-brands fa-twitter fa-3x" title="DDD South West Twitter"></i>
 		</a>
+		<a href="mailto:management@dddsouthwest.com">
+			<i class="fa-solid fa-envelope fa-3x icon" title="DDD South West Email"></i>
```

Comment:
This is the only icon with the new icon class which makes it light up slightly differently to the other icons. Was this intentional?

Reply:
No - I thought I found one which matched the rest. If I used the same classes for some reason the background went a weird purple colour